functional-tests/fn_bridge_reassignment.py

```diff
@@ -0,0 +1,107 @@
+import time
+
+import flexitest
+from strata_utils import get_balance
+
+import testenv
+from constants import UNSPENDABLE_ADDRESS
+from fn_bridge_withdraw_happy import (
+    DEPOSIT_AMOUNT,
+    OPERATOR_FEE,
+    WITHDRAWAL_EXTRA_FEE,
+    check_initial_eth_balance,
+    deposit_twice,
+    do_withdrawal,
+    setup_services,
+)
+from utils import wait_until, wait_until_with_value
+
+
+@flexitest.register
+class BridgeWithdrawReassignmentTest(testenv.StrataTester):
+    """
+    Makes two DRT deposits, then triggers the withdrawal.
+    The bridge client associated with assigned operator id is stopped.
+    After the dispatch assignment duration is over,
+    Check if new operator is being assigned or not
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        # Example: we spin up 5 operators
```

Comment:
```suggestion
        # Example: we spin up 3 operators
```

Actually, you can just remove this comment as it is very self-evident.